2_process/src/process_and_style.R

```diff
@@ -1,4 +1,5 @@
-clean_data <- function(nwis_data, nwis_info_filename){
+clean_data <- function(nwis_data_csv, nwis_info_filename){
```

Comment:
No change needed, but just wanted to note that you'll see in a lot of our project repos that we typically use more generic argument names and object names within functions. So, for example, we'd use something like `site_data_file` as the argument that we set equal to `nwis_data_csv`,  and `site_info_file` rather than the more specific `nwis_info_filename`. While this particular function is pretty specific to nwis data, we sometimes have a function that could be used to clean data from multiple sources, so we try to use variable names that are more generic and would make sense for multiple uses.